### DISCUSSION TITLE
Can clients explicitly request for a token with a shorter expiry time?

### DISCUSSION THREAD

**Discussion by makandre** [dexidp/dex#2792] [Q&A]
I have a Dex server that is using the default token expiration (i.e. 24h, and I don't necessarily have the leeway to change this)

I have a client (single page app) using this Dex and would like to get a token that expires in a shorter amount of time.  Is there a way to do this?

I'm using the authorization code flow.

Thanks.

**Comment by nabokihms** [comment]
Unfortunately but for now, there is only a global setting for the token lifetime. We may consider adding a per-connector option in the future.
